Core/Core/Api/GraphQL/Models/Version.cs

```diff
@@ -0,0 +1,20 @@
+﻿#nullable disable
+
+using System;
+
+namespace Speckle.Core.Api.GraphQL.Models;
+
+public sealed class Version
```

Comment:
I've followed our graphql schema names on all these classes, pretty much 1-1.

They do live in a clear namespace, so any naming conflicts are possible to work around.

If we start renaming classes differently from the schema, it could add a bit of confusion, but I would accept it in a few places if there were good reasons.